add-on/_locales/en/messages.json

```
@@ -750,5 +750,37 @@
   "option_telemetryGroupTracking_description": {
     "message": "Tracking description",
     "description": "A description for the 'tracking' grouping of metrics we collect (option_telemetryGroupTracking_description)"
+  },
+  "recovery_page_title" : {
+    "message": "Node Offline | IPFS Companion",
+    "description": "Title of the recovery page (recovery_page_title)"
+  },
+  "recovery_page_header" : {
+    "message": "Companion",
+    "description": "Main header on the recovery screen (recovery_page_header)"
+  },
+  "recovery_page_sub_header": {
+    "message": "Unable to reach the IPFS node :(",
+    "description": "Sub-Header on the recovery screen (recovery_page_sub_header)"
+  },
+  "recovery_page_message_p1": {
+    "message": "Make sure IPFS Desktop is running and provides the HTTP Gateway Functionality.",
```

nit: this message will be displayed also in Brave, and should not be tied to IPFS Desktop.

```suggestion
    "message": "Ensure your IPFS node runs and provides HTTP Gateway.",
```